pkg/lock/lock_debug.go

```diff
@@ -28,8 +43,89 @@ func init() {
 
```

Comment:
Maybe make `time.Second * 310` a const like `selfishThresholdSec` so they stay next to each other